pandas/io/pytables.py

```diff
@@ -1807,6 +1771,45 @@ def _read_group(self, group: "Node"):
         s.infer_axes()
         return s.read()
 
+    def _identify_group(self, key: str, append: bool) -> "Node":
+        """Identify HDF5 group based on key, delete/create group if needed."""
```

Comment:
""" on separate lines

Reply:
No problem, but is there a local policy for this? I mean according to PEP257 (https://www.python.org/dev/peps/pep-0257/#one-line-docstrings) if it fits in one line, then no need to put triple quotes on multiple lines.

Reply:
cc @datapythonista am i remembering this wrong?

Reply:
For private docstrings it doesn't make a difference, and both are correct based on Python recommendations. For public docstrings we try to standardize them to what @jbrockmendel says. We shouldn't have single liners anyway, and in our autosummaries we take the first line of the docstring (the short summary), which we want to have of similar sizes to show correctly in the html version. Having it in the line after the `"""` allows for 3 more characters, so we decided to standardize all public docstrings to that, to be consistent.